rsmtool/notebooks/comparison/header.ipynb

```diff
@@ -285,7 +305,13 @@
     "        series.append(df.apply(summary_func))\n",
     "    res = pd.concat(series, axis=1)\n",
     "    res.columns = ['MEAN', 'SD', 'MEDIAN', 'MIN', 'MAX']\n",
-    "    return res"
+    "    return res\n",
+    "\n",
+    "def color_highlighter(x, low=0, high=1, prec=3, absolute=False):\n",
+    "    abs_x = abs(x) if absolute else x\n",
```

Comment:
At least this is the one needed now. If you would want me to do that, I could move these all into a separate python program and load them in both headers.